pkgs/applications/networking/pcloud/default.nix

```
@@ -38,13 +38,13 @@
 
 let
   pname = "pcloud";
-  version = "1.14.3";
-  code = "XZ7IM70ZtWFon9pgEbk4XuvzJsTduQUyGFwV";
+  version = "1.14.4";
+  code = "XZDh750ZBgJa45xqQ8H1ztdMFX2wVhOCTOFk";
 
   # Archive link's codes: https://www.pcloud.com/release-notes/linux.html
   src = fetchzip {
     url = "https://api.pcloud.com/getpubzip?code=${code}&filename=${pname}-${version}.zip";
```

```suggestion
    url = "https://api.pcloud.com/getpubzip?code=${code}&filename=pcloud-${version}.zip";
```
https://github.com/NixOS/nixpkgs/issues/277994

---

kk, applied